cordova-lib/src/cordova/create.js

```diff
@@ -220,8 +220,22 @@ function create(dir, optionalId, optionalName, cfg) {
             }
         }
     })
-    .then(function(import_from_path) {
-
+    .then(function(input_directory) {
+        var import_from_path = input_directory;
+        var templatePkg;
```

Comment:
`templatePkg` is only used within `try` block and doesn't need to be defined in outer scope